docs/Encryption-Scheme.md

```diff
@@ -81,3 +83,88 @@ Here we need to share keys for all files inside this folder (even those, which w
 When creating the link, the creator fetches all keys currently used in the folder and creates a new keystore with it. The keystore is encrypted with a new random key. The url to the key and the encryption key are added to the link. Both, the owner and the link receiver, add this keystore to their list of keystores.
 
 Now, whenever a key for this folder is fetched, the keystore service will find the shared keystore and lookup the key there. When new files and keys are created, the keystore service also adds the key to the shared folder, so all users have access to the new file.
+
+## Different kinds of keystores
+
+We use different keystores for different purposes which will be explained briefly.
+
+
+
+#### Root - `root.json.enc`
+
+The root keystore stores the keys for all files in a person's own pod and is updated everytime a new file is created or uploaded.
+
+```json
+"keys": {
+  ...
+    "owner.solidweb.org/myCoolFile.txt":"key123",
+  ...
+}
+```
+
+#### Shared files - `shared-files.json.enc`
+
+Stores the keys for every file that has been shared by another person and is updated when opening shared links.
+
+```json
+"keys": {
+  ... 
+  "otherPerson.solidweb.org/theirCoolFile.txt":"key567",
+  ... 
+}
+```
+
+#### Shared folders - `<foldername>.json.enc`
```

Comment:
We don't use the `<foldername>`, but here we generate a random identifier: https://github.com/solidcryptpad/solidcryptpad/blob/45f30a6ca566d01f42255e318d7975ee467e7b10/src/app/services/encryption/keystore/keystore.service.ts#L139-L155

Reply:
And we (inconsistently) name it `<random-id>.shared-keystore` instead of `<random-id>.json.enc`

Reply:
will fix this in the description. should we open an issue for consistent naming?